libs/cimit-service/src/main/java/uk/gov/di/ipv/core/library/service/CiMitService.java

```diff
@@ -75,16 +76,21 @@ public CiMitService(
 
     public void submitVC(VerifiableCredential vc, String govukSigninJourneyId, String ipAddress)
             throws CiPutException {
+        String payload;
+        try {
+            payload =
+                    objectMapper.writeValueAsString(
+                            new PutCiRequest(govukSigninJourneyId, ipAddress, vc.getVcString()));
+
+        } catch (JsonProcessingException e) {
+            throw new CiPutException(e.getMessage());
```

Comment:
Do you think this message could include any sensitive information?

If you want to keep the exception details, I think we'd be better off:
- Adding a second constructor to `CiPutException` which takes a message + cause
- Making the message a generic `Failed to serialise PutCiRequest`

That way we should get the full details in the log, but only send a generic message back to the client.

(This should never happen anyway, so not worth worrying too much about...)

Reply:
Ditto below